index.html

```diff
@@ -843,7 +843,8 @@ <h2 id="BaseAudioContext">
               "#idl-def-AudioContextState.running"><code>running</code></a>
               state, the value of this attribute is monotonically increasing
               and is updated by the rendering thread in uniform increments,
-              corresponding to the audio block size of 128 samples. Thus, for a
+              corresponding to one <a href="#dfn-render-quantum">render 
```

Comment:
I think you can just say `<a>render quantum</a>`, leaving off the href stuff.  Here and every where else.